lib/ret/api/scopes.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Ret.Api.Scopes do
+  @moduledoc false
```

Comment:
I see warnings while I'm working because of an [emacs plugin](https://github.com/aaronjensen/flycheck-credo) I have installed and because we have `credo` as a dependency. I have not looked into this until now...

[`Credo`](https://github.com/rrrene/credo/) emits warnings for missing `@moduledoc` module attributes. ([Documentation](https://hexdocs.pm/credo/Credo.Check.Readability.ModuleDoc.html#content) and [Source](https://github.com/rrrene/credo/blob/b9489483b54a71f4c02d5b6579eb66668dabf1be/lib/credo/check/readability/module_doc.ex#L88))

If I run 
```
mix credo --strict
```
I see that we actually have quite a few warnings:
```
1486 mods/funs, found 7 consistency issues, 1 warning, 36 refactoring opportunities, 202 code readability issues, 54 software design suggestions.
```

I figured I'd just resolve things as I encountered them. Alternatively I can ignore the warnings, go fix all of them in a separate commit, or we can add exceptions in a credo config file. If we like `credo`, we should consider adding it as part of our automatic linting process. We already do use `mix compile --warnings-as-errors` but adding `credo` will catch even more. (I don't know how `mix compile` determines what to warn on.)

Reply:
Ah, gotcha. Yeah, that's fine then, but I agree we should have a separate PR (not just a separate commit) to fix all the credo warnings if we want to apply it to all the code.

Reply:
Ah right I meant to suggest a separate PR. I've created an issue for this. https://github.com/mozilla/reticulum/issues/431